src/tools/mail.py

```diff
@@ -0,0 +1,72 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.application import MIMEApplication
+from email.utils import formatdate
+import os
+
+class Email:
+    def __init__(self):
+        """Initialize the Email class."""
+        self.password = None
+
+    def set_password(self, password):
+        """Set the email account password."""
+        self.password = password  # Removed unnecessary 'pass'
+
+    def send_email(self, subject, body, to_email, from_email, smtp_server='smtp.gmail.com', smtp_port=587):
+        """Send a basic text email."""
+        if not self.password:
+            raise ValueError("Email password not set. Use set_password() first.")
+
+        msg = MIMEText(body)
+        msg['Subject'] = subject
```

Comment:
This would be dangerous for example what if the msg has no subject 
```suggestion
        msg['Subject'] = msg.get("Subject" , "" )
```